HW2/P4/p4

```diff
@@ -0,0 +1,20 @@
+Your writeup comparing 1, 2, and 4-way parallel performance, and discus-
+sion of your implementation choices to control cooperation between threads
+
+N = 1
+T = 3.97963690758
+
+N = 2
+T = 2.78399991989
+
+N = 4
+T = 1.81781697273
+
+N = 8
```

Comment:
Yeah, roughly same values for N=1, N=4, though my N=2 case had a significantly lower second count than yours (but was still worse than N=4).